lib/runtime/src/worker.rs

```diff
@@ -142,13 +143,13 @@ impl Worker {
 
         INIT.set(Mutex::new(Some(secondary.spawn(async move {
             // start signal handler
-            tokio::spawn(signal_handler(runtime.cancellation_token.clone()));
+            tokio::spawn(signal_handler(runtime.primary_token().clone()));
```

Comment:
Just curious - was this one a required change from making runtime private?

`runtime.cancellation_token` -> `runtime.primary_token()`

If so there might be a couple missed spots of doing the same in block_manager: https://github.com/search?q=repo%3Aai-dynamo%2Fdynamo%20runtime.cancellation_token&type=code

Reply:
And yes, it is because that field is now private.